### PR TITLE
Fixed DataSaver classes to reject outliers and increased data points

### PR DESCRIPTION
**Pull request by watakandai**

### CODE REVIEW

**Comment by watakandai** [comment]
WIP. I have not tested the script for the static and the constant data saver classes.

**Comment by watakandai** [comment]
All the scripts are running. Good to go.

**Comment by peasant98** [comment]
cool!

File: meshes/visual/arrow.dae [hunk 1]
```diff
@@ -1,132 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<COLLADA xmlns="http://www.collada.org/2005/11/COLLADASchema" version="1.4.1">
-    <asset>
```
Comment (peasant98) [inline meshes/visual/arrow.dae:3]: curious, were these arrows for the imu?

File: scripts/data_generators/ConstantRotationDataSaver.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def __init__(self, pose_names, joint_names, imu_names, filepath):
             Names of joints
         imu_names: list[str]
             Names of imus
-s        filepath: str
```
Comment (peasant98) [inline scripts/data_generators/ConstantRotationDataSaver.py:47]: nice

File: worlds/empty.world [hunk 3]
```diff
@@ -1,29 +0,0 @@
-<?xml version="1.0" ?>
-<sdf version="1.6">
```
Comment (peasant98) [inline worlds/empty.world:2]: why was this file deleted?

**Review by peasant98 — CHANGES_REQUESTED** [review]

File: meshes/visual/arrow.dae (hunk 1 above)
Comment (watakandai) [reply meshes/visual/arrow.dae:3]: it WAS. not anymore. so I'm deleting. it.

File: worlds/empty.world (hunk 3 above)
Comment (watakandai) [reply worlds/empty.world:2]: I knew it didn't work at all, so I'm gonna delete it.

**Review by peasant98 — APPROVED** [review]
lgtm.

**Pull request merged**